modin/config/envvars.py

```diff
@@ -415,38 +415,38 @@ def get(cls):
         return log_memory_interval
 
 
-class LogMemorySize(EnvironmentVariable, type=int):
-    """Size of logs (in MBs) to store per Modin job."""
+class LogFileSize(EnvironmentVariable, type=int):
+    """Max size of logs (in MBs) to store per Modin job."""
 
-    varname = "MODIN_LOG_MEMORY_SIZE"
+    varname = "MODIN_LOG_FILE_SIZE"
     default = 100
```

Comment:
Doesn't this set the default to 100 MB rather than 10MB as the doc above says?

Reply:
Yes good catch!